### PR TITLE
Move to Maven and GitHub actions.

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: TDR Deploy Auth Utils
```
Comment (TomJKing) [inline .github/workflows/deploy.yml:1]: Think the `name` value needs changing

**Review by TomJKing — COMMENTED** [review]
Looks good. One minor comment.

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (MancunianSam) [reply .github/workflows/deploy.yml]: Good point, I've changed it.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**